devel/nodejs10/Portfile

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               compiler_blacklist_versions 1.0
+
+# on macOS nodejs only builds against libc++
```

Comment:
@kencu I'd rather undo those changes. Even if node itself only builds against libc++, this may not true for native extensions, which will likely break then.